xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/server/ServerInfoLedger.java

```diff
@@ -67,15 +67,15 @@ static ImmutableServerInfoLedger.Builder builder() {
    * @return An {@link UnsignedInteger} representing the amount of XRP to reserve.
    */
   @JsonProperty("reserve_base_xrp")
-  UnsignedInteger reserveBaseXrp();
+  BigDecimal reserveBaseXrp();
 
   /**
    * Amount of XRP (not drops) added to the account reserve for each object an account owns in the ledger.
    *
    * @return An {@link UnsignedInteger} representing the amount of XRP added.
    */
   @JsonProperty("reserve_inc_xrp")
-  UnsignedInteger reserveIncXrp();
+  BigDecimal reserveIncXrp();
```

Comment:
This is a breaking change. We need to introduce the other format that @nkramer44 posted about in slack.